### PR TITLE
chore(CPlusPlus): add remove-duplicates-in-sorted-linked-list

### PR DESCRIPTION
**Pull request by SubhamKumarPandey**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] New algorithm
- [ ] Optimization in previous algorithms
- [ ] Code style update
- [ ] Refactor
- [ ] Documentation
- [ ] Other, please describe:

**Briefly describe the changes in this PR**

I had added Remove duplicated in a sorted linked list in the linked list algorithm in CPlusPlus language. It was not available there.

### CODE REVIEW

File: algorithms/CPlusPlus/Linked-Lists/remove-duplicates-in-sorted-linked-list.cpp [hunk 1]
```diff
@@ -0,0 +1,56 @@
+/*
+Traverse the list from the head (or start) node. 
+While traversing, compare each node with its next node. 
+If the data of the next node is the same as the current node then delete the next node. 
+Before we delete a node, we need to store the next pointer of the node 
+*/
```
Comment (UG-SEP) [inline algorithms/CPlusPlus/Linked-Lists/remove-duplicates-in-sorted-linked-list.cpp:6]: Add a user-friendly description of the program 
i.e
Program to remove the duplicate value in a sorted linked list using the x method

File: algorithms/CPlusPlus/Linked-Lists/remove-duplicates-in-sorted-linked-list.cpp [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+	head->next=new Node(11);
+	head->next->next=new Node(11);
+	head->next->next->next=new Node(13);
+	head->next->next->next->next=new Node(13);
+	head->next->next->next->next->next=new Node(15);
+	printlist(head);
+    removeduplicates(head);
+	printlist(head);
+	return 0;
+}
```
Comment (UG-SEP) [inline algorithms/CPlusPlus/Linked-Lists/remove-duplicates-in-sorted-linked-list.cpp:56]: add the time complexity and input-output example(test-case) at the end of the file(eof)

**Review by UG-SEP — CHANGES_REQUESTED** [review]
@SubhamKumarPandey Make the changes suggested below

File: algorithms/CPlusPlus/Linked-Lists/remove-duplicates-in-sorted-linked-list.cpp (hunk 2 above)
Comment (SubhamKumarPandey) [reply algorithms/CPlusPlus/Linked-Lists/remove-duplicates-in-sorted-linked-list.cpp:62]: @UG-SEP done the required changes

File: algorithms/CPlusPlus/Linked-Lists/remove-duplicates-in-sorted-linked-list.cpp (hunk 1 above)
Comment (SubhamKumarPandey) [reply algorithms/CPlusPlus/Linked-Lists/remove-duplicates-in-sorted-linked-list.cpp]: @UG-SEP added description

File: algorithms/CPlusPlus/Linked-Lists/remove-duplicates-in-sorted-linked-list.cpp [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    }cout<<endl;
+}
+
+//Function to Remove Duplicates from linked list
+void removeduplicates(Node *head){
+    Node *curr=head;
+    //check for duplicates if found then remove it
+    while(curr!=NULL && curr->next!=NULL){
+        if(curr->data==curr->next->data){
+            Node *temp=curr->next;
```
Comment (Ashborn-SM) [inline algorithms/CPlusPlus/Linked-Lists/remove-duplicates-in-sorted-linked-list.cpp:41]: whats the point of having the ```temp``` variable if you are not gonna use it?

File: algorithms/CPlusPlus/Linked-Lists/remove-duplicates-in-sorted-linked-list.cpp [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+            curr->next=curr->next->next;
```
Comment (Ashborn-SM) [inline algorithms/CPlusPlus/Linked-Lists/remove-duplicates-in-sorted-linked-list.cpp:42]: doing this without deallocating the ```curr->next``` node leads to memory leak.

**Review by Ashborn-SM — CHANGES_REQUESTED** [review]

File: algorithms/CPlusPlus/Linked-Lists/sadkjasdkajsdk.cpp [hunk 5]
```diff
@@ -0,0 +1,9 @@
+#include<bits/stdc++.h>
+#include <array>
+using namespace std;
+int main(){
+	std:: array<int, 5> myarray={1,2,3,4,5};
+	for(int i=0;i<array.size();i++){
+		cout<<myarray[i];
+	}
+}
```
Comment (Ashborn-SM) [inline algorithms/CPlusPlus/Linked-Lists/sadkjasdkajsdk.cpp:9]: I don't this file is relevant to the commit.
Please remove it

File: algorithms/CPlusPlus/Linked-Lists/remove-duplicates-in-sorted-linked-list.cpp (hunk 3 above)
Comment (SubhamKumarPandey) [reply algorithms/CPlusPlus/Linked-Lists/remove-duplicates-in-sorted-linked-list.cpp:41]: temp is for deallocating memory allocated to duplicate node.

File: algorithms/CPlusPlus/Linked-Lists/remove-duplicates-in-sorted-linked-list.cpp (hunk 4 above)
Comment (SubhamKumarPandey) [reply algorithms/CPlusPlus/Linked-Lists/remove-duplicates-in-sorted-linked-list.cpp:42]: Thank you I just forget to deallocate it, thanks for correcting me

File: algorithms/CPlusPlus/Linked-Lists/sadkjasdkajsdk.cpp (hunk 5 above)
Comment (SubhamKumarPandey) [reply algorithms/CPlusPlus/Linked-Lists/sadkjasdkajsdk.cpp]: This was done by mistake. Updated the changes that you have suggested and thanks for pointing out my mistakes

**Review by UG-SEP — APPROVED** [review]

**Pull request merged**